.cppcheckignore

```diff
@@ -10,4 +10,5 @@
 // as libraries/modules and are outside of the scope of this repository. 
 //
 // Another alternative is using inline suppressions. Please see Cppcheck manual
-// for further information: https://cppcheck.sourceforge.net/manual.pdf 
+// for further information: https://cppcheck.sourceforge.net/manual.pdf
+Uninitialized variable:projects/drivers/network/network.c
```

Comment:
please don't add entire files to `.cppcheckignore` unless needed
in this case you can use inline suppressions